### PR TITLE
contest ToC page for PNI

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5703

URL: https://foundation-s-pni-contes-qizech.herokuapp.com/en/privacynotincluded/contest

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/contest.html [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+          <p id="guidelines"><strong>3. Content Guidelines.</strong></p>
+          <p>
+            In addition to complying with all other requirements of these Official Rules and/or on the and/or on Twitter or Instagram;
+            each Entry must comply with the following guidelines (<strong>“Content Guidelines”</strong>). Any Entry that Sponsor, in its
+            sole discretion, determines is in violation of any Content Guideline may be disqualified.
+          </p>
+          <p>
+            For all entries:
+          </p>
+          <ul class="body">
```
Comment (mmmavis) [inline network-api/networkapi/buyersguide/templates/contest.html:126]: image

According to the mockup, this list should not have bullets. You can add `list-unstyled` to `<ul>` to unstyle it.

```suggestion
          <ul class="body list-unstyled">
```

**Review by mmmavis — CHANGES_REQUESTED** [review]
image

This list should not have bullets. You can add `list-unstyled` to `<ul>` to unstyle it.

**Review by kristinashu — APPROVED** [review]
Looks good! I'm definitely not reading through it line by line tho!

**Comment by kristinashu** [comment]
Mavis, Pomax pinged me in slack about that bullet list. It's good that it's bulleted there, it was like that in the doc but not in the comp.

**Review by mmmavis — APPROVED** [review]

File: network-api/networkapi/buyersguide/templates/contest.html (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/buyersguide/templates/contest.html:126]: it should: the original doc had them, and they were forgotten in the comp. I asked Kristina whether to omit or include them and got signoff on making sure they were there =)

**Pull request merged**